assets/scss/modules/_notifications.scss

```diff
@@ -163,7 +163,8 @@
 }
 
 .highlight-message--icon-top {
-  background-position-y: em(20);
+  background-position: em(20) em(20);
```

Comment:
so `background-position` I presume the default is in the middle? can you use `background-position: top left` and let the default padding take care of the padding?